fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -389,7 +389,16 @@ public Response createOrReplaceObjectRdf(
         }
 
         try {
-            session.save();
+            // prevent SN-Sibling nodes from creating.
+            synchronized (LOGGER) {
```

Comment:
@ajs6f Thanks. I think we can change it to use the FedoraLdp class or create a static object as the lock. What do you suggest?